### PR TITLE
[storm] fix broken and outdated links in readme

### PR DESCRIPTION
**Pull request by Davidchinacloud**
Signed-off-by: LinWengang <linwengang@chinacloud.com.cn>

### CODE REVIEW

File: staging/storm/README.md [hunk 1]
```diff
@@ -22,15 +22,15 @@ Source is freely available at:
 This example assumes you have a Kubernetes cluster installed and
 running, and that you have installed the ```kubectl``` command line
 tool somewhere in your path. Please see the [getting
-started](../../docs/getting-started-guides/) for installation
+started](https://kubernetes.io/docs/setup/) for installation
```
Comment (sebgoa) [inline staging/storm/README.md]: [getting started guides]

File: staging/storm/README.md [hunk 2]
```diff
@@ -114,7 +114,7 @@ The Storm workers need both the ZooKeeper and Nimbus services to be
 running.
 
 Use the [`examples/storm/storm-worker-controller.json`](storm-worker-controller.json) file to create a
-[replication controller](../../docs/user-guide/replication-controller.md) that manages the worker pods.
+[replication controller](https://kubernetes.io/docs/concepts/workloads/controllers/replicationcontroller/) that manages the worker pods.
```
Comment (sebgoa) [inline staging/storm/README.md]: we should really update this example to using deployments

Comment (Davidchinacloud) [reply staging/storm/README.md]: The new submission has updated  replication controller to deployment

File: staging/storm/README.md (hunk 1 above)
Comment (Davidchinacloud) [reply staging/storm/README.md]: The new submission has been modified to a more consistent links

**Pull request merged**